index.html

```diff
@@ -333,41 +346,39 @@
 								}
 							}
 							activeDistrict = coords[4];
-							if (coords[4] !== '0') {
-								for (i = 500; i <= 5500; i += 1000) {
-									setTimeout(function(){
-										updateStatsBox(filterField);
-									}, i);
-								}
-							} else {
-								document.getElementById('statsBox').style.opacity = 0;
+							for (i = 500; i <= 5500; i += 1000) {
```

Comment:
I was able to simplify this bit a little because it used to check for whether a district was selected and not run if the whole state was showing.